addons/website_forum/static/src/scss/website_forum.scss

```diff
@@ -1,223 +1,240 @@
+        > p:empty:first-child:last-child {
             display: none;
         }
-    }
 
-    .validate_answer {
-        @include o-position-absolute(0, auto, 0, 0);
-        width: 4px;
-        height: 100%;
-        z-index: 1; // On top of card content if an alert is displayed inside
+        p {
+            margin-bottom: 0.5rem;
 
-        > .bg-success {
-            @include o-position-absolute(0, 0, 0, 0);
+            > br:first-child:last-child {
+                display: none;
+            }
         }
-        > span {
-            @include o-position-absolute(10px, 50%);
-            transform: translateX(50%);
-            font-size: 20px;
+    }
 
-            &[data-href] {
-                cursor: pointer;
-            }
+    textarea.load_editor {
+        min-height: 100px;
+        opacity: 0.5;
+        background-color: gray('300');
 
-            i {
-                padding: 2px 3px;
-                border-radius: 100%;
+        + .note-editor.note-frame{
+            border-color: $border-color;
+            border-radius: $border-radius;
+            overflow: hidden;
+
+            .note-editing-area .note-editable {
+                padding-left: $input-btn-padding-x;
+                padding-right: $input-btn-padding-x;
             }
         }
     }
 
-    .user_name_people {
-        font-weight: bold;
-        max-width: 60%;
+    .o_wforum_author_box {
+        &.o_show_info {
+            line-height: 1.2;
+            img {
+                @include size(2em);
+            }
+        }
+         &.o_compact {
+            line-height: 1;
+            img {
+                @include size(1.4em);
+            }
+        }
+        .o_forum_user_info_success {
+            padding: 3px;
+            border: 2px solid;
+
+            > span {
+                @include size(1em);
+                top: 0;
+                right: -3px;
+                line-height: 0.8;
+                box-shadow: 0 0 0 2px white;
+            }
+        }
     }
 
     .oe_answer_true {
-        color: theme-color('success');
-        &:hover {
-            color: lighten(theme-color('success'), 20%);
-        }
+        @include o-hover-text-color( theme-color('success'), lighten(theme-color('success'), 20%));
     }
 
     .oe_answer_false {
-        color: $gray-400;
-        &:hover {
-            color: $gray-600;
-        }
+        @include o-hover-text-color( $gray-400, lighten(theme-color('success'), 20%));
     }
 
-    .favourite_question {
-        color: $gray-400 !important;
+    .o_wforum_follow_toggler {
+        // Reduce size
+        button, input {
+            padding: $input-btn-padding-y-sm $input-btn-padding-x-sm;
+            font-size: $input-btn-font-size-sm;
+            line-height: $input-btn-line-height-sm;
+        }
+
+        // Hide button and show 'bell' icon for logged-in users that
+        // are not followers.
+        .o_connected_user & > div[data-follow="off"] > div {
```

Comment:
can't we do it in the xml to allow easy customization ?

Reply:
Hello @JKE-be , thanks, I'll have a look... are these notes for a future review or you can work on this now?

Reply:
As explain previously, we do what we can to merge it in saas-12.5 and got it in v13.
But an 'ASAP fix' of 1325 lines of diff that change most of layout (and remove some features ?), cannot be merged just reading in 1 hour the code. But yes, we hope to find the time to merge it.

Reply:
Hi @stefanorigano , I just want to check the pr now and see that it is always in the code. 

Can you check that and try to clean the dead code.

For example you removed some class (validate_answer) , but js code is always there, your remove some event , but function (_onRemoveValidAnswerMouse) are always there, ...

And while you change the subscribe by the bell into the xml instead of css, don't you thing that we could also use the bell with another color to unsubscribe instead of this button ? 

Thanks

Reply:
`Moderation tools` need to be visible in mobile.

Reply:
Hello @JKE-be , yes, I was working on it right now.
The JS events related to 2c029a0 has been removed by mistake during  a rebase, indeed all the rest of the code (now dead) is still there, thanks to report it :) 

About 2c029a0 itself: I had already done **some**  of the the things introduced by this commit during my general REF, so I have to check what can be used/adapted and what can be removed instead.

About the "subscribe bell/button": I'll try the xpath method, as you suggested.
It may seems unnecessary but I want to  keep consistency with the "followed post" icon in the sidebar, and visually differentiate it from the favourite's "star" icon... otherwise it may be misleading. Changing icon color if active? Nice 👍

Reply:
yes, I'm with your branch running before me, and it true that most of the commit 2c029a0 is no more needed because already reimplemented by your refactoring.

about the toggle of validate_answer, we refresh to make it simple because e.g. tag 'best answer' should be moved from the old one to the new one.. what make many js for a few details, so refresh was more easy... I let you decide what's the best todo for you ;)

"Followed tags" in right columns seems broken, if you don't find the fix, let me know and I or We can check that. Actually, when you select "Followed posts" , follow post and tag are both selected.

When you try to delete a comment, the comment don't disappear for now. In Js there a code ~ $link.parents('.comment') but, there ar eno more class comment to find the commend to hide.

Reply:
Can we also re-add NEwest and Trending in the forum menu because it is probably the 2 most used button ;) now it is only available in the sort by dropdown

Can we also try to find a solution like you did for slide, about the Users submenu.
Since website_profile take the power on it, when you are on the forum and switch to the users, you have no easy way to come back on the current forum. We should find a hack similar to when you click on forum in slide, you can be back to slide easily from the menu.

Reply:
Is there a reason to don't display the sort by when you use filter Answered / Unanswered / unsolved ,  it is only available in case of all post.